.gitignore

```diff
@@ -3,3 +3,35 @@
 
 # Ignore Gradle build output directory
 build
+
+# Ignore user-specific IntelliJ files
+.idea/**/workspace.xml
```

Comment:
We should be able to ignore all of these files by just adding `.idea/`. We should also include `.classpath`, `.project`, and `.settings` since these are all IDE specific configs.

Reply:
@jayden This article said it was ok to keep .idea/ except for the files listed (https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems). Would you still like me to remove it?

I'll add .classpath, .project, and .settings to the gitignore!